buildtools/maven/maven.go

```diff
@@ -32,13 +34,24 @@ type Parent struct {
 }
 
 type Dependency struct {
-	Name    string
-	Version string
-	Failed  bool
+	Name string
+
+	GroupId    string `xml:"groupId"`
+	ArtifactId string `xml:"artifactId"`
+	Version    string `xml:"version"`
+
+	// Scope is where the dependency is used, such as "test" or "runtime".
+	Scope string `xml:"scope"`
```

Comment:
At first I was trying to not also do this in this PR because of PR #240 which addresses this question. But there's enough that I needed to change that I think that PR may not turn out to be very helpful. What do you think, do it here anyway?

Reply:
FWIW, srclib-java applies scope to each dependency it returns, and by default, the fossa UI filters to ‘compile’ and ‘runtime’ deps. You can turn other scopes on in the project settings as well. Might be worth a discussion before merging this

Reply:
Discussed this with @zlav —we decided that at least in this PR we'll focus on just discovery of modules and simply parsing the pom.xml files, and next we'll check out [#240](https://github.com/fossas/fossa-cli/pull/240) to see how best to add this feature